### PR TITLE
Add doctoc and fixed typo in README file.

### PR DESCRIPTION
**Pull request by sohankunkerkar**
/cc @abhat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +26,7 @@ This work-in-progress section describes how people _developing_ this operator ca
 ### Prerequisites
 - [Kind](https://github.com/kubernetes-sigs/kind) tool
 -  The latest version of [kubefedctl](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#kubefedctl-cli) binary installed in your $PATH 
--  If you’re planning to test against the openshift cluster then ensure that it’s already up and running.
+-  If you’re planning to test against an openshift cluster then ensure that its already up and running.
```
Comment (pmorie) [inline README.md]: "it's" is correct here - contraction of "it" and "is"

Comment (pmorie) [reply README.md]: english is weird

Comment (sohankunkerkar) [reply README.md]: Fixed.

**Comment by sohankunkerkar** [comment]
@pmorie PTAL

**Pull request merged**